### PR TITLE
Added Tests

### PR DESCRIPTION
**Pull request by dhwani07**
added more  tests for checkbox , Error Styles , Headings , Radio , Text and Text Field

### CODE REVIEW

File: src/components/ErrorText/test.cypress.tsx [hunk 1]
```diff
@@ -41,3 +41,19 @@ describe('ErrorText', () => {
...
+    cy.contains('This field is required').should('have.css', 'font-size', '16px');
+    cy.viewport(992, 1028);
+    cy.contains('This field is required').should('have.css', 'font-size', '16px');
+  });
+  it('should belong to Arial Font Family', () => {
+    mount(<TestThemeWrapper theme={crukTheme}>{content()}</TestThemeWrapper>);
+    cy.contains('This field is required')
+      .should('have.css', 'font-family')
+      .and('contain', 'Arial');
+  });
```
Comment (Develliot) [inline src/components/ErrorText/test.cypress.tsx:58]: should we be testing styles? if styles change the test breaks, isn't this the whole point of snapshots

File: src/components/Heading/test.cypress.tsx [hunk 2]
```diff
@@ -51,3 +51,173 @@ describe('Heading', () => {
...
+  });
+
+  it('should have Font weight 500 ', () => {
+    mount(<TestThemeWrapper theme={crukTheme}>{content()}</TestThemeWrapper>);
+    cy.viewport(576, 1000);
+    cy.contains('This is H6 heading').should('have.css', 'font-weight', '500');
+    cy.viewport(1200, 1028);
+    cy.contains('This is H6 heading').should('have.css', 'font-weight', '500');
+  });
+});
```
Comment (Develliot) [inline src/components/Heading/test.cypress.tsx:223]: isn't this the whole point of snapshots, 

if styles change you don't want to have to be digging into your cypress tests to fix it, you want to approve or disapprove snaps

File: src/components/Radio/test.cypress.tsx [hunk 3]
```diff
@@ -98,3 +98,15 @@ describe('Radio', () => {
...
+
+describe('Tab', () => {
+  it('Radio is tabable', () => {
+    mount(<TestThemeWrapper theme={crukTheme}>{uncontrolledRadio()}</TestThemeWrapper>);
+    cy.get('input[value=one]')
+      .focus()
+      .tab()
+      .should('have.attr', 'value', 'two')
+      .type('{downarrow}')
+      .invoke('attr', 'checked', 'true');
```
Comment (Develliot) [inline src/components/Radio/test.cypress.tsx:110]: .should('be.checked') is much nicer syntax

File: src/components/Text/test.cypress.tsx [hunk 4]
```diff
@@ -48,3 +48,20 @@ describe('Text', () => {
...
+  });
+  it('should have font size 16px', () => {
+    mount(<TestThemeWrapper theme={crukTheme}>{content()}</TestThemeWrapper>);
+    cy.contains('This is text it defaults to a paragraph').should('have.css', 'font-size', '16px');
+  });
+  it('should have font weight 500', () => {
+    mount(<TestThemeWrapper theme={crukTheme}>{content()}</TestThemeWrapper>);
+    cy.contains('Color').should('have.css', 'font-weight', '500');
+  });
+});
```
Comment (Develliot) [inline src/components/Text/test.cypress.tsx:67]: again lets not check styles, this is the whole point of snapshots

File: src/components/TextField/test.cypress.tsx [hunk 5]
```diff
@@ -143,3 +143,30 @@ describe('TextField', () => {
...
+            <TextField label="Phone number" type="text" hasError={true} name="phoneNumber" />
+          </Box>
+        }
+      </TestThemeWrapper>,
+    );
+    cy.getInputByLabel('Phone number')
+      .focus()
+      .should('have.css', 'border-color', 'rgb(213, 19, 17)');
+  });
+});
```
Comment (Develliot) [inline src/components/TextField/test.cypress.tsx:162]: again styles test should be snapshot

**Review by Develliot — CHANGES_REQUESTED** [review]

File: src/components/ErrorText/test.cypress.tsx (hunk 1 above)
Comment (dhwani07) [reply src/components/ErrorText/test.cypress.tsx:58]: makes sense -I will remove the style tests

**Pull request closed without merging**